beacon-chain/blockchain/receive_attestation.go

```diff
@@ -125,6 +120,9 @@ func (s *Service) processAttestationsRoutine(subscribedToStateEvents chan struct
 			return
 		case <-st.C():
 			s.processAttestations(s.ctx)
+			if err := s.updateHead(s.ctx, s.getJustifiedBalances()); err != nil {
```

Comment:
why not put this in `s.processAttestations` itself ?

Reply:
Sure. We also can put it after this line:
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/blockchain/receive_attestation.go#L153

I dont feel strongly about this. Would you favor that better?